test/e2e/set/scyllacluster/config.go

```diff
@@ -0,0 +1,17 @@
+package scyllacluster
+
```

Comment:
maybe rename to consts.go

Reply:
I'd prefer not to lock it to consts only, some expression types will need vars